po/ja.po

```diff
@@ -106,889 +106,889 @@ msgstr ""
 
 #: src/SUMMARY.md:31
 msgid "Slices"
-msgstr ""
+msgstr "スライス型"
 
 #: src/SUMMARY.md:32
 msgid "String vs str"
-msgstr ""
+msgstr "文字列（String） vs 文字列スライス（str）"
 
 #: src/SUMMARY.md:33
 msgid "Functions"
-msgstr ""
+msgstr "関数"
 
 #: src/SUMMARY.md:34
 msgid "Rustdoc"
-msgstr ""
+msgstr "Rustdoc"
 
 #: src/SUMMARY.md:35 src/SUMMARY.md:82
 msgid "Methods"
-msgstr ""
+msgstr "メソッド"
 
 #: src/SUMMARY.md:36
 msgid "Overloading"
-msgstr ""
+msgstr "オーバーロード"
 
 #: src/SUMMARY.md:37 src/SUMMARY.md:66 src/SUMMARY.md:90 src/SUMMARY.md:119
 #: src/SUMMARY.md:148 src/SUMMARY.md:177 src/SUMMARY.md:204 src/SUMMARY.md:225
 #: src/SUMMARY.md:251 src/SUMMARY.md:273 src/SUMMARY.md:293
 msgid "Exercises"
-msgstr ""
+msgstr "練習問題"
 
 #: src/SUMMARY.md:38
 msgid "Implicit Conversions"
-msgstr ""
+msgstr "暗黙的な型変換"
 
 #: src/SUMMARY.md:39
 msgid "Arrays and for Loops"
-msgstr ""
+msgstr "配列とfor文"
 
 #: src/SUMMARY.md:41
 msgid "Day 1: Afternoon"
-msgstr ""
+msgstr "Day 1：PM"
 
 #: src/SUMMARY.md:43
 msgid "Variables"
-msgstr ""
+msgstr "変数"
 
 #: src/SUMMARY.md:44
 msgid "Type Inference"
-msgstr ""
+msgstr "型推論"
 
 #: src/SUMMARY.md:45
 msgid "static & const"
-msgstr ""
+msgstr "static & const"
 
 #: src/SUMMARY.md:46
 msgid "Scopes and Shadowing"
-msgstr ""
+msgstr "スコープとシャドーイング"
 
 #: src/SUMMARY.md:47
 msgid "Memory Management"
-msgstr ""
+msgstr "メモリ管理"
 
 #: src/SUMMARY.md:48
 msgid "Stack vs Heap"
-msgstr ""
+msgstr "スタック vs ヒープ"
 
 #: src/SUMMARY.md:49
 msgid "Stack Memory"
-msgstr ""
+msgstr "スタックメモリ"
 
 #: src/SUMMARY.md:50
 msgid "Manual Memory Management"
-msgstr ""
+msgstr "マニュアルメモリ管理"
 
 #: src/SUMMARY.md:51
 msgid "Scope-Based Memory Management"
-msgstr ""
+msgstr "スコープによるメモリ管理"
 
 #: src/SUMMARY.md:52
 msgid "Garbage Collection"
-msgstr ""
+msgstr "ガベージコレクション"
 
 #: src/SUMMARY.md:53
 msgid "Rust Memory Management"
-msgstr ""
+msgstr "Rustのメモリ管理"
 
 #: src/SUMMARY.md:54
 msgid "Comparison"
-msgstr ""
+msgstr "比較"
 
 #: src/SUMMARY.md:55
 msgid "Ownership"
-msgstr ""
+msgstr "所有権"
 
 #: src/SUMMARY.md:56
 msgid "Move Semantics"
-msgstr ""
+msgstr "ムーブ（move）"
```

Comment:
「ムーブセマンティクス」
ムーブセマンティクス is already a common term in Japanese because it's widely used in C++ community. ([example](https://cpprefjp.github.io/lang/cpp11/rvalue_ref_and_move_semantics.html))

Reply:
noted